articles/privacy-art-collection.md

```diff
@@ -0,0 +1,35 @@
+|   |   |
+|---|---|
+| Title               | Privacy Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+↑ Add more table rows for more social media links. See [this completed article](articles/simple-payment-verification.md) for a real example.
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+Pre-discussion, research and questions
+
+---
+
+# Article Title
+
+## Art sample + Description
+
+## Art Piece 1 + Description
```

Comment:
Decred helps defending your privacy, which is very important when looking for "financial" freedom.
Shared in first place with security in my opinion
![decred defending your privacy](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/a835146b-2ae9-4d84-88ec-8c87994ca88a)

Reply:
perfect 👍

Reply:
This art is so good!